roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -269,6 +269,75 @@
     - "{{ tempdir }}/templates/logging-es-dc.yml"
     delete_after: true
 
+- name: Retrieving the cert to use when generating secrets for the {{ es_component }} component
```

Comment:
if we switch to `oc_route` we should be able to point to the file destination instead of slurping them in